packages/modules/viewer-react/src/hooks/useUiProviders.tsx

```diff
@@ -31,11 +29,14 @@ export function useUiProviders(
         })
       );
     }
-    // if (
-    //   !defaultUiConfig?.contentManipulationTools?.verticalItems?.measureTools
-    // ) {
-    //   defaultProviders.push(new MeasureToolsUiItemsProvider());
-    // }
+    if (
```

Comment:
When this is "true" it means that it should not be visible. The variable could be named better, but basically all of the ui config booleans remove content only

Reply:
discussed internally, this logic is correct, i just needed to add an extra check